macros/dmt_unit_test.sql

```diff
@@ -1,9 +1,106 @@
-{% test unit_test(model, input_mapping, expected_output, name, description, compare_columns, depends_on) %}
-    {% set test_sql = dbt_datamocktool.get_unit_test_sql(model, input_mapping, depends_on)|trim %}
-    {% do return(dbt_utils.test_equality(expected_output, compare_model=test_sql, compare_columns=compare_columns)) %}
-{% endtest %}
+{%- macro unit_test(model, input_mapping, expected_output, name, description, compare_columns, depends_on) -%}
```

Comment:
Why'd this change from a test to a macro?

Reply:
That was very sloppy from me, sorry for that. Parallelly, I tried out another idea to add support for a unit test as a test file within the tests/ folder. For that, however, I had to change the test into a macro. I haven't figured out yet, how to integrate that into the package, but maybe I'll open another PR for that later on.
For now, I simply changed it back to test.